specificatie/openapi.yaml

```diff
@@ -10,18 +10,18 @@ info:
 
     Gegevens die er niet zijn of niet actueel zijn krijg je niet terug. Had een persoon bijvoorbeeld een verblijfstitel die nu niet meer geldig is, dan wordt die verblijfstitel niet opgenomen. In partners wordt alleen de actuele of de laatst ontbonden partner geleverd.
 
-    Zie de [Features overzicht](https://brp-api.github.io/Haal-Centraal-BRP-bevragen/v2/features-overzicht) en [Getting started](https://brp-api.github.io/Haal-Centraal-BRP-bevragen/v2/getting-started) voor nadere toelichting.
+    Zie de [Features overzicht](https://developer.rvig.nl/brp-api/personen-zoeken-en-raadplegen/) en [https://developer.rvig.nl/brp-api/getting-started/) voor nadere toelichting.
   version: 2.6.0
   contact:
-    url: https://brp-api.github.io/Haal-Centraal-BRP-bevragen
+    url: https://github.com/BRP-API/Haal-Centraal-BRP-bevragen
```

Comment:
Is `https://developer.rvig.nl/brp-api/contact/` niet een betere contact verwijzing?